src/main/java/com/example/cabbooking/entity/Driver.java

```diff
@@ -0,0 +1,35 @@
+package com.example.cabbooking.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+@Entity
+@Table(name="drivers")
+public class Driver {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @Column(unique = true, nullable = false)
+    private String name;
+    private String gender;
+    private int age;
+    private boolean isAvailable;
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "vehicle_id", referencedColumnName = "id")
+    private Vehicle vehicle;
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "location_id", referencedColumnName = "id")
+    private Location loaction;
+    @OneToMany(mappedBy = "driver", cascade = CascadeType.ALL)
+    private List<Ride> ride;
+    private LocalDateTime localDateTime = LocalDateTime.now();
```

Comment:
@PrePersist
protected void onCreate() {
    createdAt = LocalDateTime.now();
}
maybe you can check if you can use this ?

Reply:
yes will check and update